src/components/Error/Error.js

```diff
@@ -0,0 +1,11 @@
+import './Error.css';
+
+function Error({error}) {
+  return (
+    <div>
+      <h1>{error}</h1>
```

Comment:
Love that you're passing a prop so that what you display can be dynamic - smart!